### PR TITLE
Import tests from SpiderMonkey

### PR DESCRIPTION
**Pull request by Ms2ger**

### CODE REVIEW

**Review by ptomato — COMMENTED** [review]
I am _definitely_ not reviewing the giant automated import. But the other stuff around it looks good.

File: tools/lint/lib/checks/includes.py [hunk 1]
```diff
@@ -43,20 +47,25 @@ def _get_includes_flow_list(source):
         return [inc.strip() for inc in match.group('includes').split(',') if inc] if match else []
 
     def run(self, name, meta, source):
+        print(name)
```
[the remark this answers is not in the document]
Comment (Ms2ger) [reply tools/lint/lib/checks/includes.py]: Yeah, exactly. Removed them now

File: features.txt [hunk 2]
```diff
@@ -84,6 +84,10 @@ Atomics.pause
 # https://github.com/tc39/proposal-is-error
 Error.isError
 
+# Records and Tuples
+# https://tc39.es/proposal-record-tuple/
+record-tuple
```
[the remark this answers is not in the document]
Comment (littledan) [reply features.txt:89]: Yeah probably these should be omitted. The idea is still that staging only includes "correct" tests.

**Review by ptomato — APPROVED** [review]
As discussed in the maintainers chat, approving and merging this.

**Comment by ptomato** [comment]
(Based on past runs, the esmeta job is likely going to time out after 6 hours, so I won't wait for that.)

**Comment by kmiller68** [comment]
Not sure where to report this but `acosh-approx.js` added with this batch breaks when the implementation just defer's to macOS Sonoma's libc++ (via `std::acosh`). Newer macOS Sequoia seems to pass though.

**Comment by nickva** [comment]
@kmiller68 noticed it as well on Sonoma with Intel arch and tried to fix in https://github.com/tc39/test262/pull/4492

**Pull request merged**